generator/osm2meta.cpp

```diff
@@ -148,14 +148,27 @@ string MetadataTagProcessorImpl::ValidateAndFormat_ele(string const & v) const
 
 string MetadataTagProcessorImpl::ValidateAndFormat_destination(string const & v) const
 {
-  return v;
+  string r;
+  strings::Tokenize(v, ";", [&](std::string_view d)
```

Comment:
What is normalized here? Is "a;b" replaced by "a; b" ?

Reply:
Yes

Reply:
Are there many badly formatted values there? Isn't string.replace() enough?

Reply:
"a; b" instead of "a;b" is typical mistake.
If you think that it will affect generation time, it can be made right before output at routing::GetFullRoadName()

Reply:
Moved to routing::GetFullRoadName()

Reply:
Why move? Initially it was in generator and let stay there.

Reply:
I want as much flexibility as possible.
As far as I understand concern of @biodranik potential problem is increased generation time.
Otherwise there is no reason to simplify formatting.
GetFullRoadName() is called is called only once per position update so it should not be a problem there.

Reply:
I don't know his concern, but definitely not generation time here. If you need to make normalization - do it. And the best place for it is generator.

Reply:
We do all normalization at the generator, to save device battery.